lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -105,8 +105,8 @@ export interface DetectionHelpers {
   isCustomQuery: IsCustomQueryFn;
   isBuiltInQuery: IsBuiltInQueryFn;
   isAsyncUtil: IsAsyncUtilFn;
-  isFireEventUtil: (node: TSESTree.Identifier) => boolean;
-  isUserEventUtil: (node: TSESTree.Identifier) => boolean;
+  isFireEventUtil: (node: TSESTree.Identifier | null) => boolean;
+  isUserEventUtil: (node: TSESTree.Identifier | null) => boolean;
```

Comment:
Could you revert all helpers here so they don't expect any `null` node, please?  So all helpers follow a similar signature, and it's also up to whichever code calling this to make sure it's expected node type before calling these helpers.

Reply:
Sure, but as you can see in 5518751, it adds a lot of noise, so I am not too sure if that's a better approach 😅